README.md

```diff
@@ -11,14 +11,35 @@ This could potentially build with earlier versions. These just happen to be the
 ones tested with locally.
 
 - gcc v13
+  - Can also use clang v16
 - make v4
 - [googletest v1.12.1](https://github.com/google/googletest/releases/tag/release-1.12.1)  (only needed for testing)
+- [LLVM v16](https://github.com/llvm/llvm-project/releases/tag/llvmorg-16.0.0)
+  - NOTE: If using a prebuilt for x86_64 linux, use the
+    [16.04](https://github.com/llvm/llvm-project/releases/tag/llvmorg-16.0.4)
+    prebuilt. The normal v16.0.0 version seems to be built incorrectly since I
+    run into a segfault with the prebuilt in
+    `llvm::ConstantExpr::getGetElementPtr(llvm::Type*, llvm::Constant*,
+    llvm::ArrayRef<llvm::Value*>, bool, std::optional<unsigned int>,
+    llvm::Type*)` but don't hit this when I build from source.
 
 ## Building + Running
 
 ```sh
-$ make
-./main
+$ make  # Creates ./main
+$ ./main test_input.txt
+; ModuleID = 'my cool jit'
+source_filename = "my cool jit"
+
+@0 = private unnamed_addr constant [16 x i8] c"Hi! I am ALIVE!\00", align 1
+
+define void @main() {
+entry:
+  %0 = call i32 @printf(ptr @0)
+  ret void
+}
+
+declare i32 @printf(ptr %0)
```

Comment:
Do we need this in the readME?

Reply:
I think it helps to show what the IR looks like. It also conveys what's getting piped to `llc` in the example code below.